hack/cmds/commands.json

```diff
@@ -5050,10 +5050,32 @@
       },
       {
         "name": "condition",
-        "type": "enum",
-        "enum": [
-          "NX",
-          "XX"
+        "type": "oneof",
+        "arguments": [
+          {
+            "name": "nx",
+            "type": "pure-token",
+            "token": "NX"
+          },
+          {
+            "name": "xx",
+            "type": "pure-token",
+            "token": "XX"
+          },
+          {
+            "type": "block",
+            "arguments": [
+              {
+                "name": "ifeq",
+                "type": "pure-token",
+                "token": "IFEQ"
+              },
+              {
+                "name": "comparison-value",
+                "type": "string"
+              }
+            ]
+          }
```

Comment:
Hi @px86, there is no need to use the `block` type, just merge the current `ifeq` and `comparison-value` into one `string` type argument.